internal/internal_workflow.go

```diff
@@ -182,6 +182,7 @@ type (
 		contextPropagators                  []ContextPropagator
 		memo                                map[string]interface{}
 		searchAttributes                    map[string]interface{}
+		parentClosePolicy                   *ParentClosePolicy
```

Comment:
Why is it a pointer? Can we always have a value?

Reply:
done